source/Calamari.AzureAppService/Calamari.AzureAppService.csproj

```diff
@@ -12,15 +12,9 @@
     <TargetFrameworks>net462;net6.0</TargetFrameworks>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Azure.Identity" Version="1.10.4" />
-    <PackageReference Include="Azure.ResourceManager" Version="1.7.0" />
```

Comment:
I figure it is better for these dependencies to be managed solely by `Calamari.Azure`